test/fixtures/input-fixture.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+
+export interface InputFixtureProps {
+    value: string;
+    onChange: (e: string) => void;
+}
+
+export interface InputFixtureState {
+    checked: boolean;
+}
+
+export class InputFixture extends React.Component<InputFixtureProps, InputFixtureState> {
```

Comment:
Please rename class, props, state and file name to contain the word "radio" since this is a radio button input